packages/system/data_stream/core/manifest.yml

```
@@ -37,5 +37,15 @@ streams:
         show_user: false
         description: >-
           Processors are used to reduce the number of fields in the exported event or to enhance the event with metadata. This executes in the agent before the logs are parsed. See [Processors](https://www.elastic.co/guide/en/beats/filebeat/current/filtering-and-enhancing-data.html) for details.
+      - name: use_performance_counters
+        type: bool
+        title: Use performance counters
+        multi: false
+        required: false
+        show_user: true
+        default: false
+        description: >-
+          This option enables the use of performance counters to collect data for cpu/core metricset. You should use this option if running beats on machins with more than 64 cores.
```

```suggestion
          This option enables the use of performance counters to collect data. You should use this option if running agent on Windows machines with more than 64 cores.
```

You can remove the windows line after this